x-pack/plugins/security_solution/public/siem_migrations/rules/components/rule_details_flyout/index.tsx

```diff
@@ -0,0 +1,250 @@
+  LARGE_DESCRIPTION_LIST_COLUMN_WIDTHS,
+} from './constants';
+import { TranslationTab } from './translation_tab';
+
+const StyledEuiFlyoutBody = styled(EuiFlyoutBody)`
+  .euiFlyoutBody__overflow {
+    display: flex;
+    flex: 1;
+    overflow: hidden;
+
+    .euiFlyoutBody__overflowContent {
+      flex: 1;
+      overflow: hidden;
+      padding: ${({ theme }) => `0 ${theme.eui.euiSizeL} 0`};
+    }
+  }
+`;
+
+const StyledFlexGroup = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+
+const StyledEuiFlexItem = styled(EuiFlexItem)`
+  &.euiFlexItem {
+    flex: 1 0 0;
+    overflow: hidden;
+  }
+`;
+
+const StyledEuiTabbedContent = styled(EuiTabbedContent)`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: hidden;
+
+  > [role='tabpanel'] {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    overflow: hidden;
+    overflow-y: auto;
+
+    ::-webkit-scrollbar {
+      -webkit-appearance: none;
+      width: 7px;
+    }
+
+    ::-webkit-scrollbar-thumb {
+      border-radius: 4px;
+      background-color: rgba(0, 0, 0, 0.5);
+      -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
+    }
+  }
+`;
+
+/*
+ * Fixes tabs to the top and allows the content to scroll.
+ */
+const ScrollableFlyoutTabbedContent = (props: EuiTabbedContentProps) => (
+  <StyledFlexGroup direction="column" gutterSize="none">
+    <StyledEuiFlexItem grow={true}>
+      <StyledEuiTabbedContent {...props} />
+    </StyledEuiFlexItem>
+  </StyledFlexGroup>
+);
+
+const tabPaddingClassName = css`
+  padding: 0 ${euiThemeVars.euiSizeM} ${euiThemeVars.euiSizeXL} ${euiThemeVars.euiSizeM};
+`;
+
+export const TabContentPadding: FC<PropsWithChildren<unknown>> = ({ children }) => (
+  <div className={tabPaddingClassName}>{children}</div>
+);
+
+interface MigrationRuleDetailsFlyoutProps {
+  ruleActions?: React.ReactNode;
+  ruleMigration: RuleMigration;
+  size?: EuiFlyoutProps['size'];
+  extraTabs?: EuiTabbedContentTab[];
+  closeFlyout: () => void;
+}
+
+export const MigrationRuleDetailsFlyout: React.FC<MigrationRuleDetailsFlyoutProps> = React.memo(
+  ({
+    ruleActions,
+    ruleMigration,
+    size = 'm',
+    extraTabs = [],
+    closeFlyout,
+  }: MigrationRuleDetailsFlyoutProps) => {
+    const { expandedOverviewSections, toggleOverviewSection } = useOverviewTabSections();
+
+    const rule: RuleResponse = useMemo(() => {
+      const esqlLanguage = ruleMigration.elastic_rule?.query_language ?? 'esql';
+      return {
+        type: esqlLanguage,
+        language: esqlLanguage,
+        name: ruleMigration.elastic_rule?.title,
+        description: ruleMigration.elastic_rule?.description,
+        query: ruleMigration.elastic_rule?.query,
+
+        // Default values
+        severity:
+          (ruleMigration.elastic_rule?.severity as Severity) ?? DEFAULT_TRANSLATION_SEVERITY,
+        risk_score: DEFAULT_TRANSLATION_RISK_SCORE,
+        from: 'now-360s',
+        to: 'now',
+        interval: '5m',
```

Comment:
Could we have all defaults in a constant? like:
```
const DEFAULT_TRANSLATION_FIELDS = {
  risk_score: 21,
  severity: 'low',
  from: 'now-360s',
  to: 'now',
  interval: '5m',
} as const;
```

And here:
```

const elasticRule = ruleMigration.elastic_rule;
{        
  type: esqlLanguage,
  language: esqlLanguage,
  name: elasticRule?.title,
  description: elasticRule?.description,
  query: elasticRule?.query,

  ...DEFAULT_TRANSLATION_FIELDS,
  ...(elasticRule?.severity && {severity: elasticRule.severity as Severity}),
  // other optional fields
}
```

Reply:
done